lib/utils/index.js

```diff
@@ -579,6 +581,55 @@ class Utils {
       throw err;
     }
   }
+
+  static iosRealDeviceUDID(udid) {
+    if (udid.length > 25) {
+      return udid;
+    }  
+    
+    if (udid.length < 24) {
+      throw new Error('Incorrect UDID provided for real iOS device');
+    }
+
+    return `${udid.substring(0, 8)}-${udid.substring(9, 25)}`;
+  };
+
+  static getSdkRootFromEnv() {
+    const androidHomeInGlobalEnv = 'ANDROID_HOME' in process.env;
+
+    dotenv.config({path: path.join(__dirname, '.env')});
+
+    const androidHome = process.env.ANDROID_HOME;
+    const fromDotEnv = androidHomeInGlobalEnv ? '' : ' (taken from .env)';
```

Comment:
Can remove `fromDotEnv` as there's actually no way to find this out in Nightwatch because env variables are already taken from .env file before `androidHomeInGlobalEnv` is assigned.